meetups/2024/2024-06-20-talk-night/program.md

```diff
@@ -0,0 +1,45 @@
+# Rust Aarhus meetup at Trifork
+
+Date: 2024-06-20
+
+Rust Aarhus invites you to this free meetup for all interested in the programming language [Rust].
+
+The program for the evening is:
+[OR   We are still working on the program, but it will be something like the following:]
+
+- 18:05 - 18:15: Welcome (10 min)
+- 18:20 - 18:50: Free slot for a talk(s)
+- 18:50 - 19:00: A word from our sponsor (5-10 min)
+- 19:00 - 19:45: Light dinner sponsored by Trifork
+- 19:45 - 20:35: Architecturing services with dependency injection by Ivan Luchev (40 min + 10 min Q&A)
+- 20:40 - 20:50: 
+- 20:50 - 21:00: Closing remarks (10 min)
+- 21:00: Let's continue the discussion at the nearby bar.
+
+## [TITLE OF FIRST TALK]
+
+By [SPEAKER]
+
+[ADD DESCRIPTION OF THE TALK HERE]
+
+[INFORMATION ABOUT THE SPEAKER]
+
+
+## Architecturing services with dependency injection
+
+By [Ivan Luchev][il]
+
+Ivan will talk about how one can architecture an app following the dependency inversion principle. We will do this by building a simple app made out of modules that interact with each other by relying on their interfaces. To achieve this, we will talk about dependency injection — how to use it, common patterns, and pitfalls.
+
+Ivan Luchev is...
```

Comment:
Ivan has been a Software Engineer at Trifork for the last 2 years, working on a Data Platform. He's coming from a background of large scale data processing with Go. His experience with Rust spans 3 years, mainly working on side projects and research.